Sources/SecureXPC/Server/ClientRequirement.swift

```diff
@@ -60,6 +60,13 @@ extension XPCServer.ClientRequirement {
             return try teamIdentifier(teamID)
         }
     }
+
+	/// The requesting client must have hardened runtime enabled.
+	public static var hardenedRuntime: XPCServer.ClientRequirement {
+		get {
+			return XPCServer.ClientRequirement(messageAcceptor:HardenedMessageAcceptor())
+		}
```

Comment:
This can be written as just:
`XPCServer.ClientRequirement(messageAcceptor: HardenedMessageAcceptor())`

Both the `get` and `return` can be made implicit.